apps/utils.py

```diff
@@ -17,6 +17,16 @@ def random_id():
     return str(uuid4())
 
 
+def is_uuid(uid, version=4):
```

Comment:
Is this used? I don't see it used anywhere in your PR diff

Reply:
You are right, this was used in the previous iteration. I didn't cancelled it because I thought we might use it down the road as a utils function.